perfkitbenchmarker/vm_util.py

```diff
@@ -47,6 +47,9 @@
 # to use this directory as a base for other module level constants.
 VM_TMP_DIR = '/tmp/pkb'
 
+# Default timeout for issuing a command.
+DEFAULT_TIMEOUT = 300
```

Comment:
I think the default should be to timeout. The only cases I can think of that we don't want to time out after waiting for 5 minutes would be long running ssh/scp commands. In every other case, if the process is still running, something has almost certainly gone wrong.